.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": "0.2.0",
```

Comment:
What's the significance of the version number?

Reply:
That's the version of the vscode config language i.e. what defines all the keys in the json. It gets autogenerated when you first create the config file.